src/components/AddMenu.js

```diff
@@ -0,0 +1,34 @@
+import { useQuery } from "react-query";
+import axios from "axios";
+
+async function fetchItems() {
+  const result = await axios.get("https://demo5940257.mockable.io/menu_items");
+  if (!result.ok) {
+    throw new Error("Network response was not ok");
+  }
+  return result.json;
+}
+
+function AddMenu() {
+  const { data, error, isError, isLoading } = useQuery(
+    "foodItems",
+    async () => await fetchItems()
```

Comment:
I'm thinking this is the preferred syntax.  No need for async/await here.

```suggestion
    () => fetchItems()
```